storage/filesystem/object.go

```diff
@@ -23,21 +23,24 @@ type ObjectStorage struct {
 	// deltaBaseCache is an object cache uses to cache delta's bases when
 	deltaBaseCache cache.Object
 
+	simpleObjectCache cache.Object
```

Comment:
Can't we reuse `deltaBaseCache`? Maybe we have to change its name to `objectCache`.

Reply:
It could definitely be reused. I wasn't sure whether that would be a preferred option. On one hand the loose object cache doesn't have to be big and serves slightly different purpose. On the other hand I assume it would be uncommon to hit both loose objects and packfile objects at the same time, so reusing the cache shouldn't hurt much.

Reply:
Then use `deltaBaseCache`. I don't expect to have a penalty using the same cache for both and makes it simpler.